driver/makedepend.ml

```diff
@@ -17,6 +17,47 @@ open Compenv
 open Parsetree
 module String = Misc.Stdlib.String
 
+module Json = struct
+  let escape_char ppf chr =
+      match chr with
+      | ('\"' | '\\') as c ->
+          Format.fprintf ppf "\\%c" c
+      | '\n' ->
+          Format.fprintf ppf "\\%c" 'n'
+      | '\t' ->
+          Format.fprintf ppf "\\%c" 't'
+      | '\r' ->
+          Format.fprintf ppf "\\%c" 'r'
+      | '\b' ->
+          Format.fprintf ppf "\\%c" 'b'
+      | '\x00' .. '\x1F' | '\x7F' as c ->
+          Format.fprintf ppf "\\u%04X" (Char.code c)
```

Comment:
We would need someone knowledgeable with JSON escaping rules to review this part.

Reply:
Strictly speaking U+007F does not need to be escaped ([ref](https://tools.ietf.org/html/rfc8259#section-79)), but it doesn't seems a bad idea to do it. The function looks fine.

Reply:
Thanks!